pam/integration-tests/testdata/TestCLIIntegration/golden/authenticate_user_switching_broker

```diff
@@ -225,9 +225,9 @@ Gimme your password
 
 
 
-Auth return: The return value should be ignored by PAM dispatch:
-Acct mgmt return: The return value should be ignored by PAM dispatch
->
+Auth user 'user-integration-switch-broker' error (25): The return value should be ignored by PAM
```

Comment:
I see that was the case too initially, but I don’t understand why we get to that state? Especially on the second call (below)

Reply:
We get this state/error message when switching to the `local` broker, which exits our PAM module. Was it supposed to be different?

Reply:
This test case refers to "user first selects an authd broker, then decides he wants to use the local broker instead".

Reply:
ok, can we create a card for renaming this test case to be more clear then and create another one which is to switch brokers to another one which is handled by authd too? Then, we can resolve this one :)

Reply:
Done!